Tools/Scripts/update-angle

```diff
@@ -121,6 +121,9 @@ cleanup_after_successful_rebase_and_exit() {
     echo "Press Enter to continue after fixing build:"
     read -r
     regenerate_changes_diff "origin/main"
+    echo "Generating contents of commit message into commit-message.txt."
```

Comment:
Could you include the standard subject, so it'd be easier to have in the commit message.

`Update ANGLE to $(git log -1 ${COMMIT_HASH} --format=%cs) (${COMMIT_HASH}))
`    

Could you format the commit message body as:

```
Contains upstream commits:
git log --oneline ${PREVIOUS_ANGLE_COMMIT_HASH}..${COMMIT_HASH} --pretty="%h %s"
$(git log --oneline ${PREVIOUS_ANGLE_COMMIT_HASH}..${COMMIT_HASH} --pretty="%h %s")

```

E.g. the git log command would be included, so it's easy to cut-paste when verifying.

Reply:
Yes, certainly - done.